tests/regression/44-trier_analyzer/02-abc.c

```diff
@@ -1,5 +1,8 @@
 #include <pthread.h>
-#include "/home/psi/nestra/kodu/mat/inf/p/teor/anal/DAEDALUS/tmp/errors.h"
+
+void err_abort(int i, char* ptr) {
+  exit(0);
+}
```

Comment:
Looks like this `err_abort` from `errors.h` was likely this macro: https://github.com/thlorenz/threads/blob/cbe6d187b82cf0a6be05245799ea02ef0a501d40/src/prog-with-posix-threads/errors.h#L38-L42. This substitute function is good though.

Reply:
Yeah, that seems to be it. I switched it back.